cpp/examples/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Copyright (c) 2021, NVIDIA CORPORATION.
+
+# libcudf examples build script
+
+# Add libcudf examples build scripts down below
+
+# Parallelism control
+PARALLEL_LEVEL=${PARALLEL_LEVEL:-4}
```

Comment:
I'm not an expert in cmake thus I'm not sure if this will result in `-j 4` or `-j -4`?

Reply:
Seems to be the same as here: https://github.com/rapidsai/cudf/blob/0b9f1789e5d5bec6a38fe57c7e4556e0dc359bdf/ci/gpu/build.sh#L17

Reply:
I got it! We will have `4`, not `-4`. The shell syntax is a bit confusing:
```
${parameter:-word}
    If parameter is unset or null, the expansion of word is substituted. 
    Otherwise, the value of parameter is substituted.
```